tests/streamlit_ui_tests/grid/keyboard_shortcuts.spec.ts

```diff
@@ -51,6 +51,31 @@ test.describe('Keyboard Shortcuts', () => {
       // with the text test
       await expect(mito.locator('.tab-selected').locator('div').filter({ hasText: "test" }).first()).toBeVisible();
     });
+
+    test('Next/prev sheet with graphs', async ({ page }) => {
+      const mito = await getMitoFrameWithTestCSV(page);
+      await importCSV(page, mito, 'test.csv');
+      await mito.getByText('Graph').click();
+      await expect(mito.getByText('Column1 bar chart')).toBeVisible();
+      await mito.locator('.tab', { hasText: 'test_1' }).click();
+      await mito.getByText('Graph', { exact: true }).click();
+      await expect(mito.getByText('Column1 bar chart')).toBeVisible();
+
```

Comment:
This test is a bit hard to parse. Confirming that we have one data tab, and two graphs after this? Comment would be nice, also saying the graph is selected.